src/Android/Accessibility/AccessibilityHelpers.cs

```diff
@@ -77,6 +77,7 @@ public static class AccessibilityHelpers
             new Browser("jp.co.fenrir.android.sleipnir_black", "url_text"),
             new Browser("jp.co.fenrir.android.sleipnir_test", "url_text"),
             new Browser("com.vivaldi.browser", "url_bar"),
+            new Browser("com.vivaldi.browser.snapshot", "url_bar"),
```

Comment:
Do we also need to add this to Autofill Helpers CompatBrowsers?

Reply:
It doesn't appear to make a difference (neither vivaldi seems to work with autofill, at least for me).  I can add it anyway if you'd like.

Reply:
Might as well add it